src/main/kotlin/racingcar/controller/CarRacing.kt

```diff
@@ -0,0 +1,91 @@
+package racingcar.controller
+
+import racingcar.model.Car
+import racingcar.model.CarList
+import racingcar.view.UserInterface
+import camp.nextstep.edu.missionutils.Randoms
+
+class CarRacing(private val view: UserInterface) {
+
+    fun run() {
+        val (cars, attempts) = handleInput()
+
+        view.showRaceStart()
+        for (i in 0 until attempts) {
+            simulateRace(cars)
+            val carNameList = cars.carList.map { it.name }
+            val carMovedList = cars.carList.map { it.distanceCovered }
+            view.printRacingCar(carNameList, carMovedList)
+        }
```

Comment:
`for` 루프를 `repeat`으로 변경하면 반복을 한다는 의미를 의도적으로 더 드러낼 수 있고, `i`와 같은 변수도 생략할 수 있어 가독성이 좋아질 것 같습니다!
```suggestion
        repeat (attempts) {
            simulateRace(cars)
            val carNameList = cars.carList.map { it.name }
            val carMovedList = cars.carList.map { it.distanceCovered }
            view.printRacingCar(carNameList, carMovedList)
        }
```
혹은 불필요한 변수 `i`를 사용하지 않는다는 뜻으로 `_` 대체하셔도 됩니다.
```
for (_ in 0 until attempts) {
    simulateRace(cars)
    val carNameList = cars.carList.map { it.name }
    val carMovedList = cars.carList.map { it.distanceCovered }
    view.printRacingCar(carNameList, carMovedList)
}
```